src/sdk.js

```diff
@@ -176,7 +183,10 @@ export default class AccountsSDK {
     }
 
     this.transaction.generate(params);
+    this.redirectUriParamsPersister.persist(params);
```

Comment:
Use `window.location.href` here and change the underlying implementation.
```suggestion
    if (preserveOriginalHref) {
       this.hrefPerister.persist(window.location.href);
    }
```

Reply:
We shouldn't rely on `window.location.href`. We should persist params directly taken from redirect_uri which can be passed by:
- sdk config
- or autorization strategy config
- or is taken from window.location.href as fallback